### DISCUSSION TITLE
copy to clipboard button

### DISCUSSION THREAD

**Discussion by mohdniyazkhan** [zerodevx/zero-md#112] [General]
As you mentioned regarding copy to clipboard button in your github document, please check the link below
https://zerodevx.github.io/zero-md/v2/?a=recipes.md
In this link I can see cdn link has been used in explaining the example, if I download the dependency in node modules, then how I can use it.
Is it possible or not?

**Comment by zerodevx** [comment]
For `v3`, add this feature like so:

https://github.com/zerodevx/zero-md/blob/f820ded7757b15a1e66c47b671088d6306ed89df/docs/index.html#L12-L48
